packages/suite/src/actions/recovery/recoveryActions.ts

```diff
@@ -85,14 +86,21 @@ const recoverDevice = () => async (dispatch: Dispatch, getState: GetState) => {
     dispatch(setError(''));
     dispatch(setStatus('in-progress'));
 
-    const response = await TrezorConnect.recoveryDevice({
+    const params: Parameters<typeof TrezorConnect['recoveryDevice']>[0] = {
```

Comment:
Looks like a weird way for defining a type. Isn't there an existing type ready to use for this?

Reply:
agree with Raf

Reply:
Well, I am not defining a type here. I am reusing an existing type imported from trezor-connect. If you have any better idea how to do it, please let me know.

Reply:
Of course I could do something like this, but it feels much less readable to me

```
 const response = await TrezorConnect.recoveryDevice({
        type: advancedRecovery ? 1 : 0,
        word_count: wordsCount,
        passphrase_protection: DEVICE.DEFAULT_PASSPHRASE_PROTECTION,
        enforce_wordlist: true,
        device: {
            path: device.path,
        },
        u2f_counter: device.features.capabilities?.includes('Capability_U2F')
            ? Math.floor(Date.now() / 1000)
            : undefined,
    });
```

Reply:
# :exploding_head: :gun:

Reply:
import { RecoveryDevice } from 'trezor-connect';

:)